attributes/default.rb

```diff
@@ -61,3 +61,46 @@
+default["confluent"]["kafka-connect"]["worker.properties"]["value.converter.schema.registry.url"] = "http://localhost:8081"
+default["confluent"]["kafka-connect"]["worker.properties"]["internal.key.converter"] = "org.apache.kafka.connect.json.JsonConverter"
+default["confluent"]["kafka-connect"]["worker.properties"]["internal.value.converter"] = "org.apache.kafka.connect.json.JsonConverter"
+default["confluent"]["kafka-connect"]["worker.properties"]["internal.key.converter.schemas.enable"] = "false"
+default["confluent"]["kafka-connect"]["worker.properties"]["internal.value.converter.schemas.enable"] = "false"
+default["confluent"]["kafka-connect"]["worker.properties"]["offset.storage.file.filename"] = "/tmp/connect.offsets"
+
+# Distributed properties only. connect-standalone works with these as well, it just appears to ignore them
+default["confluent"]["kafka-connect"]["worker.properties"]["group.id"] = "connect-cluster"
+default["confluent"]["kafka-connect"]["worker.properties"]["config.storage.topic"] = "connect-configs"
+default["confluent"]["kafka-connect"]["worker.properties"]["offset.storage.topic"] = "connect-offsets"
+
+default["confluent"]["kafka-connect"]["env_vars"] = {}
+default["confluent"]["kafka-connect"]["distributed_class"] = "org.apache.kafka.connect.cli.ConnectDistributed"
+default["confluent"]["kafka-connect"]["standalone_class"] = "org.apache.kafka.connect.cli.ConnectStandalone"
+
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.rootLogger"] = "CONSOLE,ROLLINGFILE"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.CONSOLE"] = "org.apache.log4j.ConsoleAppender"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.CONSOLE.Threshold"] = "INFO"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.CONSOLE.layout"] = "org.apache.log4j.PatternLayout"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.CONSOLE.layout.ConversionPattern"] = "[%d] %p %m (%c:%L)%n"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.ROLLINGFILE"] = "org.apache.log4j.RollingFileAppender"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.ROLLINGFILE.Threshold"] = "INFO"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.ROLLINGFILE.File"] = "/var/log/confluent/kafka-connect.log"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.ROLLINGFILE.MaxFileSize"] = "10MB"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.ROLLINGFILE.MaxBackupIndex"] = "10"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.ROLLINGFILE.layout"] = "org.apache.log4j.PatternLayout"
+default["confluent"]["kafka-connect"]["log4j.properties"]["log4j.appender.ROLLINGFILE.layout.ConversionPattern"] = "[%d] %p %m (%c:%L)%n"
+
+default["confluent"]["zookeeper"]["zookeeper.properties"]["dataDir"] = "/tmp/zookeeper"
```

Comment:
would that just be a file called attributes/_zookeeper.rb and any default attributes in it are auto-magically available?